packages/grafana-data/src/panel/PanelPlugin.ts

```diff
@@ -110,6 +111,10 @@ export class PanelPlugin<
   private suggestionsSupplier?: VisualizationSuggestionsSupplier;
 
   panel: ComponentType<PanelProps<TOptions>> | null;
+  // TODO: this creates type error in configureStore.ts not sure why
+  // explorePanel?: ComponentType<ExplorePanelProps> | null;
+  explorePanel?: any;
```

Comment:
Please don't use `any`. Maybe you can use `@ts-expect-error` or go deeper on the type error.,